CONTRIBUTING_JS.md

```diff
@@ -158,13 +126,12 @@ The commit message formatting can be added using a typical git workflow or throu
   - **feat**: A new feature
   - **fix**: A bug fix
   - **docs**: Documentation only changes
-  - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
   - **refactor**: A code change that neither fixes a bug nor adds a feature
   - **perf**: A code change that improves performance
   - **test**: Adding missing tests
   - **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
-- **Scope** - The scope could be anything specifying the place of the commit change. For example `api`, `cli`, etc...
-- **Breaking Changes** - Should be identified at the end of commit message. Start with the words `BREAKING CHANGE:` on a new line followed by a space or two new lines. The rest of the commit message is then used to describe in detail what was broken and the migration path (if there is one).
+- **Scope** - An optional scope can be added specifying the place of the commit change. For example `api`, `cli`, etc...
+- **Breaking Changes** - Should be identified by appending a `!` to the end of the type (e.g. `feat!: ...`) and at the end of commit message. Start with the words `BREAKING CHANGE:` on a new line followed by a space or two new lines. The rest of the commit message is then used to describe in detail what was broken and the migration path (if there is one).  This will appear in the generated release notes.
```

Comment:
```suggestion
- **Breaking Changes** - Should be identified by appending a `!` to the end of the type (e.g. `feat!: ...`). Start with the words `BREAKING CHANGE:` on a new line followed by a space or two new lines. The rest of the commit message is then used to describe in detail what was broken and the migration path (if there is one).  This will appear in the generated release notes.
```